staff/robert/task-app/task-app-client/src/task-api-client.js

```diff
@@ -0,0 +1,39 @@
+const axios = require('axios');
+
+const URL = `http://localhost:5000/api`
+
+function getReq (path) {
+    return axios.get(path)
```

Comment:
MAY you do axios.get(path).then(res => res.data) to return the clean data in the next then.

Reply:
done